frontend/nuxt.config.ts

```diff
@@ -127,20 +116,18 @@ export default defineNuxtConfig({
     apiClientSecret: "",
     public: {
       // These values can be overridden by the NUXT_PUBLIC_* env variables
-      deploymentEnv,
-      apiUrl,
+      deploymentEnv: "local",
+      apiUrl: "https://api.openverse.org/",
       providerUpdateFrequency: 3600000,
       savedSearchCount: 4,
       sentry: {
         dsn: "",
-        environment: deploymentEnv,
-        release: import.meta.env.SEMANTIC_VERSION,
+        environment: "local",
+        release: "",
```

Comment:
Release should be kept reading from `import.meta.env.SEMANTIC_VERSION`, it is definitively a build-time variable. If there's a way to configure it outside of runtime config in order to make that unambiguously clear, we should. It must not be set at runtime (it makes no sense to do so).